pipelines/julia_sdm.json

```diff
@@ -0,0 +1,444 @@
+        "target": "126",
+        "targetHandle": "background",
+        "id": "reactflow__edge-125background-126background"
+      },
+      {
+        "source": "34",
+        "sourceHandle": "clean_presence",
+        "target": "126",
+        "targetHandle": "occurrence",
+        "id": "reactflow__edge-34clean_presence-126occurrence"
+      },
+      {
+        "source": "126",
+        "sourceHandle": "sdm",
+        "target": "128",
+        "targetHandle": null,
+        "id": "reactflow__edge-126sdm-128"
+      },
+      {
+        "source": "126",
+        "sourceHandle": "sdm_uncertainty",
+        "target": "129",
+        "targetHandle": null,
+        "id": "reactflow__edge-126sdm_uncertainty-129"
+      },
+      {
+        "source": "126",
+        "sourceHandle": "fit_stats",
+        "target": "130",
+        "targetHandle": null,
+        "id": "reactflow__edge-126fit_stats-130"
+      },
+      {
+        "source": "134",
+        "sourceHandle": null,
+        "target": "96",
+        "targetHandle": "bbox",
+        "id": "reactflow__edge-134-96bbox"
+      },
+      {
+        "source": "134",
+        "sourceHandle": null,
+        "target": "139",
+        "targetHandle": "bbox",
+        "id": "reactflow__edge-134-139bbox"
+      },
+      {
+        "source": "139",
+        "sourceHandle": "predictors",
+        "target": "34",
+        "targetHandle": "predictors",
+        "id": "reactflow__edge-139predictors-34predictors"
+      },
+      {
+        "source": "139",
+        "sourceHandle": "predictors",
+        "target": "125",
+        "targetHandle": "predictors",
+        "id": "reactflow__edge-139predictors-125predictors"
+      },
+      {
+        "source": "139",
+        "sourceHandle": "predictors",
+        "target": "126",
+        "targetHandle": "predictors",
+        "id": "reactflow__edge-139predictors-126predictors"
+      },
+      {
+        "source": "140",
+        "sourceHandle": null,
+        "target": "96",
+        "targetHandle": "proj",
+        "id": "reactflow__edge-140-96proj"
+      },
+      {
+        "source": "34",
+        "sourceHandle": "clean_presence",
+        "target": "141",
+        "targetHandle": null,
+        "id": "reactflow__edge-34clean_presence-141"
+      },
+      {
+        "source": "125",
+        "sourceHandle": "background",
+        "target": "142",
+        "targetHandle": null,
+        "id": "reactflow__edge-125background-142"
+      },
+      {
+        "source": "144",
+        "sourceHandle": null,
+        "target": "96",
+        "targetHandle": "taxa",
+        "id": "reactflow__edge-144-96taxa"
+      }
+    ],
+    "inputs": {
+      "data>pyLoadObservations>pyLoadObservations.yml@96|data_source": {
+        "description": "Source of the data (One of gbif_pc - Planetary computer or gbif_api - GBIF Download API)",
+        "label": "Data source",
+        "type": "options",
+        "options": [
+          "gbif_pc",
+          "gbif_api"
```

Comment:
You should put gbif_api as a constant for now, latest news there was a bug with the PC and we were waiting for a fix from them.